Android/app/build.gradle

```
@@ -125,11 +129,12 @@ dependencies {
     implementation 'androidx.constraintlayout:constraintlayout:2.1.2'
     implementation "androidx.viewpager2:viewpager2:1.0.0"
     // For Firebase Analytics, etc.
-    implementation 'com.google.firebase:firebase-analytics:19.0.2'  // Last version for API <19
-    implementation 'com.google.firebase:firebase-perf:20.0.4'
-    implementation 'com.google.firebase:firebase-crashlytics:18.2.6'
-    implementation 'com.google.firebase:firebase-crashlytics-ndk:18.2.6'
-    implementation 'com.google.firebase:firebase-config:21.0.1'
+    implementation 'com.google.firebase:firebase-analytics:22.1.2'  // Last version for API <19
```

I think this comment is not accurate any more?

```suggestion
    implementation 'com.google.firebase:firebase-analytics:22.1.2'
```